glib/src/clone.rs

```diff
@@ -12,6 +12,7 @@ use std::{
 pub trait Downgrade
 where
     Self: Sized,
+    Self::Weak: Upgrade
```

Comment:
Why not `type Weak: Upgrade;` below?

Reply:
Because I didn't knew you can write it like this!